airbyte-config/specs/src/main/java/io/airbyte/config/specs/SeedConnectorSpecGenerator.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.config.specs;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.google.cloud.storage.StorageOptions;
+import com.google.common.annotations.VisibleForTesting;
+import io.airbyte.commons.io.IOs;
+import io.airbyte.commons.json.Jsons;
+import io.airbyte.commons.util.MoreIterators;
+import io.airbyte.commons.yaml.Yamls;
+import io.airbyte.config.DockerImageSpec;
+import io.airbyte.config.EnvConfigs;
+import io.airbyte.protocol.models.ConnectorSpecification;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.CommandLineParser;
+import org.apache.commons.cli.DefaultParser;
+import org.apache.commons.cli.HelpFormatter;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.Options;
+import org.apache.commons.cli.ParseException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
```

Comment:
good comment! mention fetching from the gcs bucket and that it will fail if it's not there and _why_ we made that choice.

Reply:
Done 👍

Reply:
i think mentioning specifically the choice we made here to _only_ pull from the bucket cache is worth mentioning. someone is going to come along and not understand why we did it. i think there's value in saying, since this is happening at build time, we depend on the bucket cache instead of needing to run docker containers during the build which could be slow and unwieldy. if there is a failure look in the bucket cache and figure out how to get the spec for the connector in there.

Reply:
That makes sense, I have updated the comment

Reply:
nice comment